Automattic/jetpack
# Backup: Transition from showing Plan and Connection status to relying on My Jetpack

Removes Plan and Connection status secions to favour the ones shown by My Jetpack 

#### Changes proposed in this Pull Request:

* Removes usage of MyPlan
* Removes MyPlan component file
* Removes usage of connection Status card and related code from the `useConection` hoook

#### Does this pull request change what data or activity we track or use?



#### After
![image](https://user-images.githubusercontent.com/746152/154757605-92e43ac4-48da-4c69-af8a-1be8b1df3024.png)

![image](https://user-images.githubusercontent.com/746152/154799353-39dd655e-a046-4f8f-88e5-9cc3892d7faa.png)

![image](https://user-images.githubusercontent.com/746152/154799497-a1a157bb-93f1-4c9a-abed-059c95adfdcd.png)


#### Before

![image](https://user-images.githubusercontent.com/746152/154799539-3fb18626-ea93-40b7-b9cb-1966d9940216.png)




#### Testing instructions:
* Checkout this branch or [launch a Jurassic Ninja site this branch for the Backup plugin](https://jurassic.ninja/create?jetpack-beta&branches.jetpack-backup=update/backup-for-my-jetpack&wp-debug-log&my-jetpack&no-jetpack)
* Go to the Backup menu item under Jetpack.
* Connect the site . No need for purchasing
* Back into the Backup page, confirm all looks good
